src/steps/zsh.rs

```diff
@@ -55,6 +55,24 @@ pub fn run_zplug(base_dirs: &BaseDirs, run_type: RunType) -> Result<(), Error> {
     run_type.execute(zsh).args(&["-c", cmd.as_str()]).check_run()
 }
 
+pub fn run_zplugin(base_dirs: &BaseDirs, run_type: RunType) -> Result<(), Error> {
+    let zsh = require("zsh")?;
+    let zshrc = zshrc(base_dirs).require()?;
+
+    env::var("ZPLGM[HOME_DIR]")
+        .map(PathBuf::from)
+        .unwrap_or_else(|_| base_dirs.home_dir().join(".zplugin"))
+        .require()?;
+
+    print_separator("zplugin");
+
+    let cmd = format!(
+        "aource {} && zplugin self-update && zplugin update --all",
```

Comment:
You probably meant `source`

Reply:
Oops! Thanks for catching this.  Probably happened when I was hastily fixing it in vim.